libs/api/ebpf_api.cpp

```diff
@@ -1406,6 +1406,7 @@ clean_up_ebpf_program(_In_ _Post_invalid_ ebpf_program_t* program) noexcept
     ebpf_free(program->program_name);
     ebpf_free(program->section_name);
     ebpf_free((void*)program->log_buffer);
+    ebpf_free((void*)program->hash_type);
```

Comment:
All of these changes in this file are unnecessary.

Reply:
Removed as suggested.